src/scripts/update-icons.js

```diff
@@ -116,6 +116,22 @@ const mutateIcons = () => {
   }
 }
 
+const generateMeta = () => {
+  const icons = fs
+    .readdirSync(FINAL_FOLDER)
+    .filter((f) => f.endsWith('.svg'))
+    .map((f) => path.basename(f, '.svg'))
+    .reduce((prev, next) => ({ ...prev, [next]: next }), {})
```

Comment:
Why a map where the key and value are identical and not an array?

Reply:
So originally I was going to just use an array but I thought it might be a nice easy way of discovering icons:

```ts
import { icons } from '@brave/leo/icons/meta'
import Icon from '@brave/leo/react/icon'

export default function() {
    return <div>
        <Icon name={icons.check}/>
    </div>
}
```